op-chain-ops/genesis/check.go

```diff
@@ -181,8 +181,16 @@ func PostCheckUntouchables(udb state.Database, currDB *state.StateDB, prevRoot c
 		log.Info("checked code hash", "address", addr, "hash", hash)
 
 		// Ensure that the current/previous roots match
-		prevRoot := prevDB.StorageTrie(addr).Hash()
-		currRoot := currDB.StorageTrie(addr).Hash()
+		prevStorage, err := prevDB.StorageTrie(addr)
```

Comment:
It is possible for the trie (first arg) to be `nil` if the account is not in the trie. I ran into this when working on 4337 stuff. I don't think we should ever hit the `nil` case, but best to check for `nil` to prevent an unexpected panic. If the trie is nil then its `Hash()` is `EmptyStorageRoot`